Plugins/Json/include/Acts/Plugins/Json/VolumeBoundsJsonConverter.hpp

```diff
@@ -0,0 +1,51 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Geometry/VolumeBounds.hpp"
+#include "Acts/Plugins/Json/ActsJson.hpp"
+
+#include <array>
+#include <vector>
+
+// Custom Json encoder/decoders. Naming is mandated by nlohmann::json and thus
+// can not match our naming guidelines.
+namespace Acts {
+
+static std::vector<std::string> volumeBoundTypes = {
```

Comment:
adding `const` might be good. I think this could be `constexpr` if you use `std::array<std::string_view>` but maybe that's overkill